clients/python/girder_client/cli.py

```diff
@@ -154,16 +155,21 @@ def format_options(self, ctx, formatter):
                    '[default: %s]' % GirderClient.DEFAULT_API_ROOT,
               show_default=True,
               cls=_AdvancedOption)
-@click.option('--ssl-verify/--no-ssl-verify', default=True,
-              help='Enable or disable SSL Verification',
+@click.option('--no-ssl-verify', is_flag=True, default=False,
```

Comment:
It is usually better to avoid double negation. That is why having the variable named `ssl_verify` instead of `no_ssl_verify`

Reply:
Would it be simpler to just have a flag named `--insecure`, similar to curl?

Reply:
> the positive is the default and only the negative needs to be specified.

This is why the following was working:

```
@click.option('--ssl-verify/--no-ssl-verify', default=True,
              help='Enable or disable SSL Verification',
```

Default: ssl-verify is True

Then, if --no-ssl-verify is passed .. it is False.

I suggest you keep the previous flag, and simply add the certfile support.